server/worker/worker.py

```diff
@@ -4,6 +4,10 @@
 from server.worker.tasks import run_new_tasks
 from server.sentry import configure_sentry
 
+# We have to import all of the api so that the modules load and all of the task
+# handlers get registered as background_tasks.
+from server import api  # pylint: disable=unused-import
```

Comment:
This was happening implicitly before through the import of `server.sentry`, which accidentally was importing all the api through a series of imports. I just happened upon this and decided to make it explicit